doc/template.html

```diff
@@ -4,8 +4,9 @@
   <meta charset="utf-8">
   <title>__SECTION__ Node.js __VERSION__ Manual &amp; Documentation</title>
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic">
+  <link href='https://fonts.googleapis.com/css?family=Fira+Mono:400,700' rel='stylesheet' type='text/css'>
```

Comment:
Can be removed. I was tinkering with bold source code. That looked good, but just too much w/ 700. 

Will reorder. That is how google fonts gives you the link.